### PR TITLE
Fix spanish language

### PR DESCRIPTION
**Pull request by jgtorcal**
Fix some spanish translations.

https://github.com/FreshRSS/FreshRSS/issues/6322

### CODE REVIEW

**Comment by math-GH** [comment]
Awesome! Thanks! All leftovers are translated. It is great to have a full Spanish translation for the upcoming 1.24.0 release.

**Review by math-GH — APPROVED** [review]

File: app/i18n/es/admin.php [hunk 1]
```diff
@@ -187,33 +187,33 @@
...
 		'apply' => 'Aplicar',
-		'changelog' => 'Changelog',	// TODO
+		'changelog' => 'Changelog',	// IGNORE
 		'check' => 'Buscar actualizaciones',
-		'copiedFromURL' => 'update.php copied from %s to ./data',	// TODO
+		'copiedFromURL' => 'update.php copiado desde %s a ./data',
 		'current_version' => 'Dispones de la versión',
 		'last' => 'Última comprobación',
-		'loading' => 'Updating…',	// TODO
+		'loading' => 'Actualizando...',
```
Comment (Alkarex) [inline app/i18n/es/admin.php]:
```suggestion
		'loading' => 'Actualizando…',
```

Unicode typography

File: app/i18n/es/sub.php [hunk 2]
```diff
@@ -127,61 +127,61 @@
 				'xpath' => 'XPath para:',
 			),
 			'json_dotnotation' => array(
-				'_' => 'JSON (dot notation)',	// TODO
+				'_' => 'JSON (sin notación)',
```
Comment (Alkarex) [inline app/i18n/es/sub.php]: Could you please double check this translation?

Comment (jgtorcal) [reply app/i18n/es/sub.php]: I fixed it

File: app/i18n/es/admin.php (hunk 1 above)
Comment (jgtorcal) [reply app/i18n/es/admin.php]: In Spanish the simple point is used.

Comment (Alkarex) [reply app/i18n/es/admin.php]: Please see https://es.wikipedia.org/wiki/Puntos_suspensivos#Uso_en_computadoras

Comment (jgtorcal) [reply app/i18n/es/admin.php]: My apologies, this is the first time I know about this. I have learned something new. 

Thank you.

Fixed in 37e1a18 commit.

**Comment by Alkarex** [comment]
Thanks 👍🏻 
Please add a line for you in https://github.com/FreshRSS/FreshRSS/blob/edge/CREDITS.md

**Pull request merged**